### PR TITLE
docs: enable live reload during editing

### PR DESCRIPTION
**Pull request by ableuler**
The question asked by @cchoirat reminded me that the setup I use for editing docs might be useful for others too:

```
rerun --ignore _build 'make html' &
httpwatcher --root ./_build/html/ 
```
gives you live updates in your browser when editing the docs. If people think it's an overkill I can keep this for myself, otherwise I will add a bit of documentation to it.

### CODE REVIEW

**Comment by rokroskar** [comment]
I usually just rebuild the docs and reload the file... so this will update as you type?

**Comment by ableuler** [comment]
> I usually just rebuild the docs and reload the file... so this will update as you type?

Depending on your editor you might have to save, but essentially yes.

**Comment by ableuler** [comment]
> actually I take it back - could you add some words to this effect in the developer docs?

sure, that was the plan.

File: docs/developer/contributing/documentation.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. _documentation:
+
+Contributing to Documentation
```
Comment (rokroskar) [inline docs/developer/contributing/documentation.rst]:
```suggestion
Contributing to the Documentation
```

Comment (rokroskar) [reply docs/developer/contributing/documentation.rst]: Oops sorry about the underline...

**Pull request merged**